service/controller/kvm/v17/resource/configmap/resource.go

```diff
@@ -56,15 +60,23 @@ func New(config Config) (*Resource, error) {
 		return nil, microerror.Maskf(invalidConfigError, "%T.RegistryDomain must not be empty", config)
 	}
 
+	for _, subnet := range config.ControlPlaneSubnets {
+		_, _, err := net.ParseCIDR(subnet)
+		if err != nil {
+			return nil, microerror.Maskf(invalidConfigError, "%T.ControlPlaneSubnets must consist valid IP subnets", config)
```

Comment:
I like the check here.
Idea: Could it make sense to have a log which shows the subnet which is used? I can imagine that it would be helpful to know what the operator handles internally while debugging. This log would only appear during start up.

My reason for this is that the parsing from installations is not super trivial through the different layers and clarity while debugging is always nice.

Reply:
Which logger would I use then here? `config.Logger`?

Reply:
There is only one `ControlPlaneSubnets` parameter. It accepts from 1 to 3 values. So it is not that hard to identify those subnets. Otherwise, can you suggest logging approach here? I saw `config.Logger.LogCtx` used only in some integration tests, but not in the production code.

Reply:
`config.Logger.LogCtx` is fine to me. 

Your call if it is really necessary.